packages/experience-builder-sdk/src/hooks/useBreakpoints.ts

```diff
@@ -26,6 +28,8 @@ const toCSSMediaQuery = ({ query }: Breakpoint): string | undefined => {
   return undefined;
 };
 
+const AvailableDesignTokenVariables = new Set(['cfPadding', 'cfMargin']);
```

Comment:
The uppercase first letter is standard for component names and types/interfaces but uncommon for variable names.
```suggestion
const availableDesignTokenVariables = new Set(['cfPadding', 'cfMargin']);
```